data/857/657/71/85765771.geojson

```diff
@@ -88,13 +115,15 @@
     "wof:lang":[
         "spa"
     ],
-    "wof:lastmodified":1582381931,
+    "wof:lastmodified":1705624223,
     "wof:name":"Seguro Social",
     "wof:parent_id":-1,
     "wof:placetype":"neighbourhood",
     "wof:repo":"whosonfirst-data-admin-cr",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        1126088885
```

Comment:
Similar `1126088885` should be in the PR but isn't...

Reply:
I don't see 1126088885 in this repo at all? Is it in another repo? Yes!

- https://github.com/whosonfirst-data/whosonfirst-data-admin-xx/blob/master/data/112/608/888/5/1126088885.geojson

So please create a PR in `whosonfirst-data-admin-xx` repo and link it to this PR.

Reply:
Same deal, this one lives in XX